src/firewall/core/nftables.py

```diff
@@ -212,6 +212,10 @@ def _icmp_types_fragments(protocol, type, code=None):
         "ttl-zero-during-transit": _icmp_types_fragments("icmpv6", "time-exceeded", 0),
         "unknown-header-type": _icmp_types_fragments("icmpv6", "parameter-problem", 1),
         "unknown-option": _icmp_types_fragments("icmpv6", "parameter-problem", 2),
+        "mld-listener-query": _icmp_types_fragments("icmpv6", "mld-listener-query"),
+        "mld-listener-report": _icmp_types_fragments("icmpv6", "mld-listener-reprot"),
+        "mld-listener-done": _icmp_types_fragments("icmpv6", "mld-listener-done"),
+        "mld2-listener-report": _icmp_types_fragments("icmpv6", "mld2-listener-report"),
```

Comment:
nit: please add them sorted in the list.

Reply:
done.